Sources/Swagger/Operation.swift

```diff
@@ -11,9 +11,9 @@ public struct Operation {
     public let operationParameters: [PossibleReference<Parameter>]
 
     public var parameters: [PossibleReference<Parameter>] {
-        return operationParameters + pathParameters.filter { pathParam in
-                operationParameters.contains { $0.value.name == pathParam.name }
-        }
+        return pathParameters.filter { pathParam in
+            !operationParameters.contains { $0.value.name == pathParam.value.name }
+        } + operationParameters
```

Comment:
Great stuff! Looks like there were 2 issues here. 
Is there any particular reason why `operationParameters` is moved to after `pathParameters`? My reasoning was that the common ones should be at the front